mode/src/processing/mode/android/Manifest.java

```diff
@@ -48,7 +48,7 @@
   static final String MANIFEST_ERROR_MESSAGE =
     "Errors occurred while reading or writing " + MANIFEST_XML + ",\n" +
     "which means lots of things are likely to stop working properly.\n" +
-    "To prevent losing any data, it's recommended that you use “Save As”\n" +
+    "To prevent losing any data, it's recommended that you use Save As\n" +
```

Comment:
Save As was highlighted please revert the change here.

Reply:
Yes, I don't know why but an unknown error was faced on this line due to these two characters used to highlight. After removing these characters Gradle's build was successful. That can be somewhat related to IDE saving the file in unknown encoding. I will confirm if the same error encounters again.

I am reverting the changes for now as you suggested. Thanks for pointing it out!